platform/ui/index.js

```diff
@@ -1,22 +1,6 @@
 import * as utils from './src/utils/';
-import {
-  ModalProvider,
-  ModalConsumer,
-  useModal,
-  withModal,
-} from './src/contextProviders';
+export * from './src/components';
+export * from './src/contextProviders';
```

Comment:
While this may be less code, it's also less clear (at least to me) what we're exposing as public when we use `*`

Reply:
It makes sense, I agree. I'll fix this file to expose all the exports explicitly.